notebooks/00_GettingStarted.ipynb

```diff
@@ -1248,8 +1248,9 @@
         "model = BaseModel.from_pretrained(\"mpariente/DPRNNTasNet-ks2_WHAM_sepclean\")\n",
         "\n",
         "# You can pass a NumPy array:\n",
-        "mixture, _ = sf.read(\"female-female-mixture.wav\", dtype=\"float32\")\n",
-        "model.separate(mixture)\n",
+        "mixture, _ = sf.read(\"female-female-mixture.wav\", dtype=\"float32\", always_2d=True)\n",
+        "mixture = mixture.T[None]\n",
+        "out_wavs = model.separate(mixture)\n",
```

Comment:
```suggestion
        "mixture, _ = sf.read(\"female-female-mixture.wav\", dtype=\"float32\", always_2d=True)\n",
        "# Soundfile returns the mixture as shape (time, channels), and Asteroid expects (batch, channels, time).\n",
        "mixture = mixture.reshape(1, mixture.shape[1], mixture.shape[0])\n",
        "out_wavs = model.separate(mixture)\n",
```

Reply:
Note I just coded this up in the GitHub editor so maybe there is a syntax error.

Maybe we could also drop the batch dimension here, not sure if that's easier to understand for the reader.

Reply:
On second thought, the code above only works if channels is 1, so if we decide to ignore batch here, maybe we should just use `mixture.transpose()` instead?

Reply:
> On second thought, the code above only works if channels is 1, so if we decide to ignore batch here, maybe we should just use `mixture.transpose()` instead?

If 'separate' is not used in the training steps, I think using `mixture.transpose()` is easier. Otherwise, I think showing the users a consistent shape of the mixture is better, because if they want to train, they have to know the batch dimension. (I'm new to Asteroid, I don't know if 'separate' is called in the training steps, hahaha...)

Reply:
I think the `mixture.reshape` is very explicit, that's good, let's go with that.